src/lib/rc/sbus.cpp

```diff
@@ -231,6 +286,32 @@ bool
 sbus_input(int sbus_fd, uint16_t *values, uint16_t *num_values, bool *sbus_failsafe, bool *sbus_frame_drop,
 	   uint16_t max_channels)
 {
+#if defined(__PX4_POSIX_OCPOC)
```

Comment:
Alright, it looked great in flight, no issues with control. Thanks for pointing this out; it seems that's been an unnecessary change all along.